adagucserverEC/CDataPostProcessors/CDataPostProcessor.cpp

```diff
@@ -37,6 +38,7 @@ CDPPExecutor::CDPPExecutor() {
   dataPostProcessorList->push_back(new CDPPOperator());
   dataPostProcessorList->push_back(new CDPPWFP());
   dataPostProcessorList->push_back(new CDPPWindSpeedKnotsToMs());
+  dataPostProcessorList->push_back(new CDPPSolarTerminator());
```

Comment:
Can you also:
- Write a news item in news.md
- Update the version number in the dockerfile
- Update the version number in Definitions.h?

Reply:
I would say this is a minor as it is a very "niche" type of functionality and not something in the core working of Adaguc?

Reply:
I put 2.28.2